cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/Committee.hs

```diff
@@ -207,17 +216,42 @@ hprop_golden_governance_committee_cold_extended_key_signing =
 hprop_golden_governance_committee_hot_extended_key_signing :: Property
 hprop_golden_governance_committee_hot_extended_key_signing =
   propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
-    skeyFile <- noteInputFile "test/cardano-cli-golden/files/input/governance/committee/cc.extended.hot.skey"
+    void $ noteInputFile "test/cardano-cli-golden/files/input/governance/committee/cc.extended.hot.skey"
     txBody <- noteInputFile "test/cardano-cli-golden/files/input/governance/drep/extended-key-signing/tx.body"
 
     outGold <- H.note "test/cardano-cli-golden/files/golden/governance/committee/tx.hot.extended.signed"
     outFile <- H.noteTempFile tempDir "outFile"
 
-    H.noteM_ $ execCardanoCLI
+    H.noteM_ $ bracketSem extendedHotCCSkeySem $ \skeyFile -> execCardanoCLI
       [  "conway", "transaction", "sign"
       , "--tx-body-file", txBody
       , "--signing-key-file", skeyFile
       , "--out-file", outFile
       ]
 
-    H.diffFileVsGoldenFile outFile outGold
\ No newline at end of file
+    H.diffFileVsGoldenFile outFile outGold
+
+-- | Execute me with:
+-- @cabal test cardano-cli-golden --test-options '-p "/golden verification key committee/"'@
+hprop_golden_verification_key_committee :: Property
+hprop_golden_verification_key_committee = do
+  let values = [ ( extendedHotCCSkeySem -- Contains the path to the skey file
+                 , "test/cardano-cli-golden/files/golden/governance/committee/cc.extended.hot.vkey"
+                 )
+                 ,
+                 ( extendedColdCCSkeySem -- Contains the path to the skey file
+                 , "test/cardano-cli-golden/files/golden/governance/committee/cc.extended.cold.vkey"
+                 )
+               ]
+
+  propertyOnce $ forM_ values $ \(semaphore, vkeyGolden) ->
+    H.moduleWorkspace "tmp" $ \tempDir -> do
+      vkeyFileOut <- noteTempFile tempDir "cc.extended.vkey"
+
+      H.noteM_ $ bracketSem semaphore $ \skeyFile -> execCardanoCLI
+        [  "conway", "key", "verification-key"
+        , "--signing-key-file", skeyFile
```

Comment:
You don't need a semaphore for the signing key. It's used as an input here, so protecting it seems redundant.